src/main/java/frc/robot/commands/lightstrip/EnabledLight.java

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+/*
+ * Asimov's Laws:
+ * The First Law: A robot may not injure a human being or, through inaction, allow a human being to come to harm.
+ * The Second Law: A robot must obey the orders given it by human beings except where such orders would conflict with the First Law.
+ * The Third Law: A robot must protect its own existence as long as such protection does not conflict with the First or Second Law.
+ */
+
+package frc.robot.commands.lightstrip;
+
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj.DriverStation.Alliance;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.Constants;
+import frc.robot.RobotContainer;
+import frc.robot.subsystems.AddressableLightStrip;
+import java.util.Optional;
+
+/** Activates the light strip when the robot is enabled. */
+public class EnabledLight extends Command {
+  private AddressableLightStrip robotLights;
+
+  /** Creates a new RobotEnabledLights. */
+  public EnabledLight() {
+    this.robotLights = RobotContainer.robotLights;
+    addRequirements(this.robotLights);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    Optional<Alliance> alliance = DriverStation.getAlliance();
```

Comment:
We should really just store alliance stuff in Constants, but this is fine for now. (do not change anything here)